lib/Runtime/Library/JavascriptString.cpp

```diff
@@ -2107,15 +2098,6 @@ namespace Js
         JavascriptString* pThis = nullptr;
         GetThisStringArgument(args, scriptContext, L"String.prototype.toUpperCase", &pThis);
 
-        // Fast path for one character strings
```

Comment:
Not able to justify the merit (or lack of) of the fast path right now. Therefore I believe it is prudent to leave the fast path intact and only swap the ad-hoc CharToLowerCase/CharToUpperCase to WINAPI. Updated in the latest rev.